components/wb-bubble/wb-bubble-doc-en.html

```diff
@@ -0,0 +1,124 @@
+		<p>When executing a such action can make your page not WCAG compliant.</p>
+	</div>
+</div>
+
+<h2>Working examples</h2>
+<ul>
+	<li>English: <a href="bubble-en.html">Quick action bubble with default avatar - opening a link</a></li>
+	<li>French: <a href="bubble-fr.html" lang="fr">Bulle liée à une action rapide avec l'avatar par défaut - overture d'un lien</a></li>
+	<li>English: <a href="bubble-en.html">Virtual assistant bubble with customized avatar - opening an overlay</a></li>
+	<li>French: <a href="bubble-fr.html" lang="fr">Bulle  d'assistant virtuel avec un avatar personnalisé - overture d'une fenêtre superposée</a></li>
+</ul>
+
+<h2>How to implement</h2>
+<ul>
+	<li>Add the <code>wb-bubble</code> class to a <code>&lt;button&gt;</code> or a <code>&lt;a&gt;</code> element in order to apply the bubble pattern.</li>
+	<li>When using the <code>&lt;button&gt;</code> element, add and set the <code>data-wb-doaction</code> attribute in order to open the overlay wet pluggin.</li>
+	<li>When using the link <code>&lt;a&gt;</code> element, set the <code>href</code>attribute to the page as direct link..</li>
+</ul>
+
+<h2>Evaluation and report</h2>
+<p>There is no evaluation and report available for this component.</p>
+
+<h2>API (Version 1.0)</h2>
+<table class="table table-bordered">
+	<tr>
+		<th>CSS Class</th>
+		<th>Template</th>
+		<th>Visual rendering</th>
+		<th>Schema</th>
+	</tr>
+	<tr>
+		<td>Version 1.0</td>
+		<td>Version 1.0</td>
+		<td>Version 1.0</td>
+		<td>n.a.</td>
+	</tr>
+</table>
+
+<h3>CSS Class (v1.0)</h3>
+<dl class="dl-horizontal">
+	<dt><code>wb-bubble</code></dt>
+	<dd>Component: Quick action bubble</dd>
+</dl>
+<details>
+	<summary>Version notes</summary>
+	<dl class="dl-horizontal">
+		<dt>Version 1.0 (v8.0)</dt>
+		<dd>
+			<ul>
+				<li>Created: <code>.wb-bubble</code></li>
+			</ul>
+		</dd>
+	</dl>
+</details>
+
+<h3>Template (v1.0)</h3>
+<p>Notes:</p>
+<ul>
+	<li>Can only be used with a button (<code>&lt;button&gt;</code>) to open an overlay or with a link element (<code>&lt;a&gt;</code>) to open a page.</li>
+</ul>
+
+<pre>
+	<code>&lt;button data-wb-doaction='{ &quot;action&quot;: &quot;openDialog&quot;, &quot;source&quot;: &quot;#mid-screen&quot; }' class=&quot;wb-bubble&quot;&gt;&lt;/button&gt;</code>
+</pre>
+<p>Where:</p>
+<dl class="dl-horizontal">
+	<dt><code>&quot;action&quot;: &quot;openDialog&quot;</code></dt>
+	<dd>Will open an overlay by using the overlay plugin.</dd>
+	<dt><code>&quot;source&quot;: #mid-screen</code></dt>
+	<dd>The source represent a pointer to section id, which will be included in the overlay panel</dd>
+</dl>
+
+
+
+<h3>Template (v1.0)</h3>
+<p>Notes:</p>
+<ul>
+	<li>Can be used on a button element along with the <code>data-wb-doaction</code> to open an overlay in a context of a virtual assistant. </li>
+	<li>Can be used in a link element to open action a link to a page.<li>
+</ul>
```

Comment:
You seem to have doubled the Template section here, I believe this one should be about the visual rendering instead.